Obsidian/Commands/MainCommandModule.cs

```diff
@@ -331,6 +331,38 @@ public Task StopAsync(CommandContext ctx)
         return Task.CompletedTask;
     }
 
+    [Command("ban")]
+    [CommandInfo("Bans a user.", "/ban <user> <duration>")]
+    [RequirePermission(permissions: "obsidian.ban")]
+    public async Task BanAsync(CommandContext ctx, string username, int duration)
+    {
+        var server = (Server)ctx.Server;
+        var player = server.GetPlayer(username);
+        server.Bans.AddBan(player, duration);
+        await player.KickAsync($"{username} was banned by an Operator");
```

Comment:
It would be better to specify the operator's nickname than just the operator (or if it was console/rcon then "Console")
same goes to other commands

Reply:
not a bad idea

Reply:
+1